vowpalwabbit/sample_pdf.cc

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
+// individual contributors. All rights reserved. Released under a BSD (revised)
+// license as described in the file LICENSE.
+
+#include "sample_pdf.h"
+#include "err_constants.h"
+#include "api_status.h"
+#include "debug_log.h"
+#include "parse_args.h"
+#include "../explore/explore.h"
```

Comment:
Do not use relative includes like this. Explore should be available through how the target is defined in CMake.

Reply:
Fixed.

Reply:
include not needed.  Removed.